api/cache.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/labstack/echo"
+	"github.com/tsuru/rpaas-operator/rpaas"
+)
+
+func cachePurge(c echo.Context) error {
+	var args rpaas.PurgeCacheArgs
+	err := c.Bind(&args)
+	if err != nil {
+		return err
+	}
+	name := c.Param("instance")
+	if len(name) == 0 {
+		return c.String(http.StatusBadRequest, "instance is required")
+	}
+	manager, err := getManager(c)
+	if err != nil {
+		return err
+	}
+	count, err := manager.PurgeCache(c.Request().Context(), name, args)
+	if err != nil {
+		return err
+	}
+	return c.String(http.StatusOK, fmt.Sprintf("Path found and purged on %d servers", count))
```

Comment:
When no objects were purged, does it make sense saying "Path found"?

Reply:
I think so, the message says "Path found and purged on 0 servers" when no objects are purged. However, I agree that we can return a better message. What do you think?

Reply:
I mean saying "Path found" makes no sense. What about only to say "Object purged on %s servers".

Reply:
Yeah, I think that makes a lot more sense. I'll change the message